src/eventtimes.jl

```diff
@@ -39,3 +39,9 @@ end
 
 isevent(ev::EventTime) = ev.status
 iscensored(ev::EventTime) = !ev.status
+
+## StatsModels compatibility
+
+StatsModels.concrete_term(t::Term, xs::AbstractVector{<:EventTime}, ::Nothing) =
+    StatsModels.ContinuousTerm(t.sym, first(xs), first(xs), first(xs), first(xs))
```

Comment:
Why not actually compute the mean, variance, min and max?

Reply:
Is that even defined for event times?  How would the truncation be handled?

Maybe a better option is to add methods like `mean(<:AbstractVector{<:EventTime})`?

Reply:
Hard to tell... But it also sounds weird to pass identical values. Maybe `nothing` should be allowed?